### DISCUSSION TITLE
Informations for Pro plans

### DISCUSSION THREAD

**Discussion by D0wn3r** [viewflow/viewflow#438] [General]
We are considering financing you, as we use it internally and that the free version helped us a lot, but we have some concerns we wish to have an answer from you :

- Do you have a better documentation with the pro version ?
- Can you explain us the differents plans you have for viewflow. We would like to be able to manage job with celery and also use your FSM part

**Comment by D0wn3r** [comment]
up !

**Comment by kmmbvnr** [comment]
PRO version provides celery.Job node for background jobs processing

The only documentation available is for PRO version only. There is no open-source version documented.

PRO version customers have access to private support over helpdesk tickets.

**Comment by kmmbvnr** [comment]
If you have more questions about PRO, please contact me directly over corporate email to mikhail@viewflow.io
